src/main/java/meditracker/library/LibraryManager.java

```diff
@@ -2,18 +2,19 @@
 
 import meditracker.ui.Ui;
 
-import java.io.File;
 import java.io.FileNotFoundException;
+import java.io.InputStream;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Objects;
 import java.util.Scanner;
 
 /**
  * Manages the searching of medications in the medication library.
  */
 public class LibraryManager {
 
-    private static final String FILE_PATH = "./data/medicationLibrary.txt";
+    private static final String FILE_PATH = "/medicationLibrary.txt";
```

Comment:
Would it be better to specify the FILE_PATH like so:
```suggestion
    private static final String FILE_PATH = "./medicationLibrary.txt";
```
The period signifies a relative file path which is generally recommended.